### DISCUSSION TITLE
projects

### DISCUSSION THREAD

**Discussion by wofuername** [community/community#3159] [Mobile] [answered]
i wanna have my projects everywhere so also projects and i think it's very important to have the projects also in the app

**Comment by candyho** [comment, marked as answer]
Hey thanks for your feedback! Feel free to upvote on this topic related to Projects to help boost awareness of this request! - https://github.com/github/feedback/discussions/3023 
Also appreciate if you can share more details with us in the thread above 👆 - how would you expect to manage projects on mobile? Do you expect to view the full kanban board on mobile? What are the key project features that you think we should cater on mobile?
